### PR TITLE
added stories and changed footer image

### PR DESCRIPTION
**Pull request by nfraser92**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -3,3 +3,9 @@ li {
     display: inline;
     padding: 5px 5px;
 }
+
```
Comment (MoSilvera) [inline styles.css:27]: I have some things on this line already!

File: newsFeed.html [hunk 2]
```diff
@@ -19,14 +19,52 @@
...
...3fyWKQy8izerLmdhqFZxmmny5DPkV6jKTVxojil1lZ2ErnzYwcpb8se//wAMW9gvM74x8GdZ3e+G8F+pT2T6/wB/k4JRaZ60ZJo7fSfKjSi44Tayyrqd/hNRay+mxyqvpR2TZapXfEm5LLxhe3uC3VFexU1C89XCu7aZnXN1z+SzfJPMsrOcYKFenj8MMUhG2RSqkLedx5rDAkVSJtjSkQ5CmwB0TbHyOgR0YAYw2RGMSwrNcnguUL9/q3M4NCuKZk2jcp1FJZQZiUqrXItK9fsReL6KLIvZfkwOIrK8WN0FSrqW3IHVh7IkciORN5YLpmCQMFsndEbyR0wGOxkJsSLEwkOCmWban+p8ly+TJWCTpWSUKPWX0QriplpB8RWctxmkQUnJ2yWrPJWyHNkQGNFFy1kSVVuitbPct1dx47RGepWVZP1MsUWQS33+4dKobwxpK4l7JXqyGVT3AlJDuVojGNMrzNfTL7KUZc0/usGRUIozaeVs0c8lZ243R10qSlJNcs/OEkQN8OemG/qslWz1Tii84i1hJd285Cuqucvq+n5ItHQmV6tfMsv+37FerPL+mB5ywmvt+f8Aggb/ANh0gWNV5/QiqSHlIikxkhGwWxIQhxRIcYcxhCEIABRDQ0R2zGFxD5ByIxghJjCMAu0blvbJLxy7mang0KM8xz/eSUo0Uix3KXcFyfcJgsCCzMYkLIiwo6RccuFJFWmt18hV3uFE57pEs5bESCjul8gzCJH6HkAEhMAw9N7l3OxQyW6Eth4PZLKvZHNYeUR+b3RPVKkwS0ww2iTzkN5yIRAsr0RI55BY8EScAAWkQFyF2+v3K7gu42yA4jqZYVTOF8hXLilFqSbecrs8lZSAmCguTCnIjyPwj8IwARDuIuExhkx8jqI+AGsEJIfAsmBY4IzFgxgkhxkOYwwhMRjDMtWM98d/3KjChLDTA1oKNbAHCBxguRKihmjiEWFJKP8AUh63MYQfQklsVF7j1OYhBFa2Mh4sQgWGh3EOnLAhGsWrRLJ5K9SPukIQzYsVRHhe7/A6a7DiFsrVj8Y2RCNZuqFwhcAwjWChcIsDCMYWB+EQjGaH4QWhCAahYEIRg0MMIRg0MOhCMah8jiEA1AsSEIJqBYhCMYvUp5SHYhC0Oj//2Q==" alt="flu shot pic" class="news-feed-pics">
```
Comment (MoSilvera) [inline newsFeed.html]: Add this image to your repo in order to make the link more manageable

**Review by MoSilvera — DISMISSED** [review]
make the size of the link more manageable, move the css to a lower line so there are no merge conflicts

File: styles.css (hunk 1 above)
Comment (nfraser92) [reply styles.css:27]: 👍

File: newsFeed.html (hunk 2 above)
Comment (nfraser92) [reply newsFeed.html]: :shipit:

**Pull request closed without merging**